proto/umee/oracle/v1/query.proto

```diff
@@ -93,6 +93,38 @@ service Query {
     option (google.api.http).get =
         "/umee/historacle/v1/denoms/median_deviations";
   }
+
+  // MedianOfMedians returns the median of a specified amount of
+  // stamped medians
+  rpc MedianOfMedians(QueryMedianOfMedians)
+      returns (QueryMedianOfMediansResponse) {
+    option (google.api.http).get =
+        "/umee/historacle/v1/denoms/median_of_medians";
+  }
+
+  // AverageOfMedians returns the average of a specified amount of
+  // stamped medians
+  rpc AverageOfMedians(QueryAverageOfMedians)
+      returns (QueryAverageOfMediansResponse) {
+    option (google.api.http).get =
+        "/umee/historacle/v1/denoms/average_of_medians";
+  }
+
+  // MinOfMedians returns the min of a specified amount of
+  // stamped medians
+  rpc MinOfMedians(QueryMinOfMedians)
+      returns (QueryMinOfMediansResponse) {
+    option (google.api.http).get =
+        "/umee/historacle/v1/denoms/min_of_medians";
+  }
+
+  // MaxOfMedians returns the max of a specified amount of
+  // stamped medians
+  rpc MaxOfMedians(QueryMaxOfMedians)
+      returns (QueryMaxOfMediansResponse) {
+    option (google.api.http).get =
+        "/umee/historacle/v1/denoms/max_of_medians";
+  }
```

Comment:
Why do we expose this methods? Who is going to use it? As noted in https://github.com/umee-network/umee/pull/1632#discussion_r1043840468, these methods should be avoided, unless really needed.

Reply:
We could keep them out of the grpc queries, but still expose on the Keeper for leverage to use

Reply:
Yeah they still need to be exposed for x/leverage. Iirc these queries were intended to be useful when debugging, but if they're too expensive we can just leave them out

Reply:
I'm good with that. I'm gonna take those out and keep the update on the Medians query for this PR.

Reply:
> We could keep them out of the grpc queries

I thought that's the plan. But my preference is in fact to implement it as a method over the series of prices (as is done in the `util` module), and return sequence, and let the client (leverage module) do whatever calculation it needs without risking that someone will call `Max` and `Min` within the same transaction for example. Again, see https://github.com/umee-network/umee/pull/1632#discussion_r1043840468